docker-compose.yml

```diff
@@ -19,9 +19,9 @@ services:
     ports:
       - "127.0.0.1:8355:80"
 
-  pretix:
-    image: eventyay/eventyay-tickets:development
-    container_name: eventyay-tickets
+  eventyay-ticket:
+    image: eventyay/eventyay-ticket:development
+    container_name: eventyay-ticket
```

Comment:
rename service name, and use singular ticket name for docker image
NEEDS CHANGES in the deployment